help/sources/tutorials/ui/create/customer-success/zendesk.md

```diff
@@ -0,0 +1,141 @@
+---
+keywords: Experience Platform;Zendesk;sources;connectors;source connectors;sources sdk;sdk;SDK
+title: Create a Zendesk source connection in the UI
+description: Learn how to create a Zendesk source connection using the Adobe Experience Platform UI.
+hide: true
+hidefromtoc: true
+exl-id: 
+---
+# Create a Zendesk source connection in the UI
+
+This tutorial provides steps for creating a Zendesk source connector using the Platform user interface.
+
+## Overview
+
+[Zendesk](https://www.zendesk.com) is a popular customer service solution and sales tool.
+
+This [!DNL Adobe Experience Platform] [!DNL sources connector](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) leverages the [!DNL Zendesk Search API > Export Search Results](https://developer.zendesk.com/api-reference/ticketing/ticket-management/search/#export-search-results) that returns users information into Experience Platform from Zendesk for further processing.
+
+Zendesk uses bearer tokens as an authentication mechanism to communicate with the Zendesk API.
+
+## Prerequisites
+
+The following items are required before you start configuring the extension:
+
+* You need to have a Zendesk Support account. 
+    * Go to the Zendesk [register](https://www.zendesk.com/register/) page to register and create a Zendesk account, if you do not have one already.
+
+### Gather required credentials
+
+In order to connect Zendesk to Platform, you must provide values for the following connection properties:
+
+| Credential | Description | Example |
+| --- | --- | --- |
+| subdomain | Unique domain specific to your account created during the registration process. | *xxxxx.zendesk.com*|
+| API token | Zendesk API token.<br></br>Navigate to the Zendesk website and access **[!UICONTROL Settings]** > **[!UICONTROL Apps and Integrations]** > **[!UICONTROL Zendesk API]** page. Then Navigate to the **[!UICONTROL API Tokens]** section. |*0lZnClEvkJSTQ7olGLl7PMhVq99gu26GTbJtf*|
+
+![Zendesk domain](../../../../images/tutorials/create/zendesk/zendesk-domain.png?lang=en)
```

Comment:
We can probably just skip this domain screenshot and just add a link to "Zendesk website" in the credential tables